ApiDocs.DocumentationGeneration/SampleJsonGenerator.cs

```diff
@@ -0,0 +1,90 @@
+﻿// ---------------------------------------------------------------------------
+//  <copyright file="SampleJsonGenerator.cs" company="Microsoft Corporation">
+//     Copyright © Microsoft Corporation. All rights reserved.
+//  </copyright>
+// ---------------------------------------------------------------------------
+
+namespace ApiDocs.DocumentationGeneration
+{
+    using System;
+    using System.Collections.Generic;
+
+    using ApiDocs.Validation.OData;
+
+    using Newtonsoft.Json.Linq;
+
+    public static class SampleJsonGenerator
```

Comment:
Did you look and merging this with ODataParser.BuildJsonExample()? It would be nice to have a single class responsible for generating example Json data in the project.